### PR TITLE
Add passing dotnet version in public

### PR DESCRIPTION
**Pull request by DrewScoggins**
This will get the dotnet version that the runtime repo uses and use that version
to install the dotnet. This ensures that we will always be on the same version
and stop getting failures from bad builds being pushed.

### CODE REVIEW

File: eng/pipelines/coreclr/templates/run-performance-job.yml [hunk 1]
```diff
@@ -55,7 +55,7 @@ jobs:
...
         - IsInternal: -Internal
       - ${{ if ne(parameters.osGroup, 'Windows_NT') }}:
-        - HelixPreCommand: 'sudo apt-get -y install python3-venv;python3 -m venv $HELIX_WORKITEM_PAYLOAD/.venv;source $HELIX_WORKITEM_PAYLOAD/.venv/Scripts/activate;export PYTHONPATH=;pip3 install --user azure.storage.blob==12.0.0 --force-reinstall;pip3 install --user azure.storage.queue==12.0.0 --force-reinstall;export PERFLAB_UPLOAD_TOKEN="$(PerfCommandUploadTokenLinux)"'
+        - HelixPreCommand: 'sudo apt-get install jq;sudo apt-get -y install python3-venv;python3 -m venv $HELIX_WORKITEM_PAYLOAD/.venv;source $HELIX_WORKITEM_PAYLOAD/.venv/Scripts/activate;export PYTHONPATH=;pip3 install --user azure.storage.blob==12.0.0 --force-reinstall;pip3 install --user azure.storage.queue==12.0.0 --force-reinstall;export PERFLAB_UPLOAD_TOKEN="$(PerfCommandUploadTokenLinux)"'
```
Comment (lpereira) [inline eng/pipelines/coreclr/templates/run-performance-job.yml]: `jq` is installed, but not used there.  Also, might be a better idea to install with `-y`, or coalesce with the other `apt-get` invocation (apt-get takes some time to start).

Comment (DrewScoggins) [reply eng/pipelines/coreclr/templates/run-performance-job.yml]: Yeah, I ended up realizing that 😄 I ended up just using python to do the parsing.

Comment (DrewScoggins) [reply eng/pipelines/coreclr/templates/run-performance-job.yml]: Also just using this to test the PR, I will make the check-in over in Arcade

**Comment by stephentoub** [comment]
@DrewScoggins, can this be closed?

**Comment by DrewScoggins** [comment]
Yeah, we fixed this another way and I forgot to close this PR.

**Pull request closed without merging**